be/src/exec/pipeline/operator.h

```diff
@@ -249,6 +254,9 @@ class OperatorFactory {
     std::vector<ExprContext*> _runtime_in_filters;
     std::shared_ptr<RefCountedRuntimeFilterProbeCollector> _runtime_filter_collector;
     std::vector<SlotId> _filter_null_value_columns;
+    // Mappings from input slot to output slot of ancestor exec nodes (include itself).
+    // It is used to rewrite runtime in filters.
+    std::vector<TupleSlotMapping> _tuple_slot_mappings;
```

Comment:
I think only project_operator need this _tuple_slot_mappings？

Reply:
@kangkaisen  The `ProjectOperator` must not have in filters, and the child of `ProjectOperator` may have in filters. 
Therefore, the child need `_tuple_slot_mappings`.